packages/good-design/src/core/buttons/ActionButton.tsx

```diff
@@ -0,0 +1,35 @@
+
+import { StyleSheet } from "react-native";
+import { Button, IButtonProps } from "native-base";
+import React from "react";
+
+interface IBasicButtonProps extends IButtonProps {
+	text: string
+}
+
+export const ButtonAction = ({ text, ...props }: IBasicButtonProps) => {
+	return <Button { ...props } style={styles.buttonAction}>{text}</Button>
+}
+
+const styles = StyleSheet.create({
```

Comment:
why aren't we using nativebase styles?

Reply:
This has been discussed and is being worked on as part of #16 
and there is an example app https://github.com/johnsmith-gooddollar/GoodDesignNativeBase
where Alex has defined a general default flow for so that we have consistent usage of nativebase/styling
is still to be tested

Reply:
so ClaimButton comments should be done in #16
features got mixed during fixing of the branches last week or prior